examples/update-on-async-rendering/updating-state-from-props-after.js

```diff
@@ -0,0 +1,29 @@
+// After
+class ExampleComponent extends React.Component {
+  // Initialize state in constructor,
+  // Or with a property initializer.
+  // highlight-next-line
+  state = {};
+
+  // highlight-next-line
+  static getDerivedStateFromProps(
+    nextProps,
+    prevState
+  ) {
+    // highlight-range{1-11}
+    if (
+      nextProps.currentRow !==
+      prevState.lastRow
+    ) {
+      return {
+        lastRow: nextProps.currentRow,
+        isScrollingDown:
+          nextProps.currentRow >
+          prevState.lastRow,
+      };
```

Comment:
Very aggressive line breaks make this hard to read. Could we increase the allowed width just a tiny bit?

<img width="668" alt="screen shot 2018-02-07 at 08 12 19" src="https://user-images.githubusercontent.com/810438/35905349-aa7de744-0bde-11e8-9984-b484651df32e.png">

Reply:
Yeah, man I feel the same about this. I set the line widths I did for Prettier because of mobile layout, but it really makes the desktop layout suffer in cases like this.

I'll play with it a bit to see if I can't find a better solution (maybe smaller font on smaller screens to offset the extra characters a bit).

Reply:
I think that's probably best done as a follow-up. Even though we don't have many examples, changing the Prettier width affects most of them (which also impacts their relative line highlights).

Maybe I can do it as a stacked PR, on top of this one.

Reply:
Can't we apply this on master once, and later reapply on top of this PR?

Reply:
Yeah I mean, I guess we could. It's probably fine though?

Reply:
Whatever you prefer :-) I generally prefer doing unrelated changes separately so that they don't depend on each other. This way your final blog post PR won't have to touch every single example. But it's up to you really.

Reply:
The Prettier change only took a minute to do. I just wanted to make sure we were both happy with the resulting impact on mobile. Since it impacted this PR's contents _way more_ than master, I did it on top of this PR.

When I actually merge it, maybe I'll do it directly on master and rebase. I dunno. At the moment, I'm just concerned about getting the content of this PR right. 😄